extension/src/popup/views/Preferences/index.tsx

```diff
@@ -74,7 +74,7 @@ export const Preferences = () => {
               <AutoSaveFields />
               <div className="Preferences--section">
                 <div className="Preferences--section--title">
-                  {t("Verification with")} stellar.expert
+                  {t("Verification with")} stellar.expert & blockaid.io
```

Comment:
Since we're using both stellar.expert and blockaid, I think it's appropriate to add this to the settings field. I'll have product weigh in